docs/installation/linux/debian.md

```diff
@@ -36,6 +36,26 @@ Docker is supported on the following versions of Debian:
 
      $ uname -r
 
+ Additionally, for users of Debian Wheezy, backports must be available. To enable backports in Wheezy:
+ 
+ 1. Log into your machine and open a terminal with `sudo` or `root` privileges.
+ 
+ 2. Open the `/etc/apt/sources.list.d/backports.list` file in your favorite editor.
+
+     If the file doesn't exist, create it.
+ 
+ 3. Remove any existing entries.
+ 
+ 4. Add an entry for backports on Debian Wheezy.
+
+     An example entry:
+
+    deb http://http.debian.net/debian wheezy-backports main
+
+ 5. Update package information:
+
+         $ apt-get update
```

Comment:
Thanks, sorry, working with sporadic Internet today. Your instructions need
to say it's a force push rather than commit FYI. Thanks for the tip.

On Thu, 26 May 2016, 12:09 Sebastiaan van Stijn, <notifications@github.com>
wrote:

> In docs/installation/linux/debian.md
> <https://github.com/docker/docker/pull/22999#discussion_r64728038>:
>
> > +
> > + 2. Open the `/etc/apt/sources.list.d/backports.list` file in your favorite editor.
> > +
> > +     If the file doesn't exist, create it.
> > +
> > + 3. Remove any existing entries.
> > +
> > + 4. Add an entry for backports on Debian Wheezy.
> > +
> > +     An example entry:
> > +
> > +    deb http://http.debian.net/debian wheezy-backports main
> > +
> > + 5. Update package information:
> > +
> > +         $ apt-get update
>
> couldn't agree more.
>
> can you address my earlier comment #22999 (comment)
> <https://github.com/docker/docker/pull/22999#discussion_r64660048>, then
> I think we're ready to merge 💃
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/22999/files/5fa9b8db7fb04425f77ef97bfee826cd40259efd#r64728038>
>